src/dynamicprompts/wildcardmanager.py

```diff
@@ -21,9 +21,9 @@ def _is_relative_to(p1: Path, p2: Path) -> bool:
         return False
 
 
-def _clean_wildcard(wildcard: str) -> str:
+def _clean_wildcard(wildcard: str, *, wildcard_wrap: str) -> str:
     wildcard = (
-        wildcard.strip("_")  # remove wildcard delimiters
+        wildcard.strip(wildcard_wrap)  # remove wildcard delimiters
```

Comment:
This should use `removesuffix` + `removeprefix` instead, since the wrap can be multicharacter.